test/markup/bash/arithmetic.txt

```diff
@@ -6,3 +6,10 @@ echo $(( 16#$yumi / 1003 ))
 B=20
 yumi=deadbeef
 echo $(( $B#$yumi / 1003 ))
+
+(( 19 << 42 ))
```

Comment:
```suggestion

# test that << within an expression doesn't false positively as a heredoc...
(( 19 << 42 ))
```

Reply:
THis context is helpful later.  And the expect will need to be updated.  Otherwise looks great!